pkg/rolie/clone.go

```diff
@@ -98,6 +99,8 @@ func (f *fetcher) getRemoteResourceRaw(URI string) (io.ReadCloser, error) {
 	if err != nil {
 		return nil, err
 	}
+	// Send GolieVersion in Header
+	req.Header.Set("Golie-Version", version.Version)
```

Comment:
What about sending standard http `User-agent` request header? https://en.wikipedia.org/wiki/User_agent

Reply:
looking into it now

Reply:
Are we planning on modifying the default user-agent being sent already? I added some debug just to print out the HEADER and USER-AGENT being passed:
```
INFO[2020-07-10T09:20:59-04:00] Received GET request from 127.0.0.1:64472 for ROLIE Feed at ../examples/rolie/feed/gov.nist.nvd.cve.recent.xml 
INFO[2020-07-10T09:20:59-04:00] HEADER_DUMP::map[Accept:[application/json] Accept-Encoding:[gzip] Golie-Version:[v0.1.0] User-Agent:[Go-http-client/1.1]] 
INFO[2020-07-10T09:20:59-04:00] USERAGENT_DUMP::Go-http-client/1.1 
```

I just want to make sure I understand your take on the user-agent.. if you would like to add send a custom one that is fine, we can figure it out.

Reply:
Correct. I would went ahead and rewrite default user agent.

It holds no information. As soon as you see `golie-xyz`, you can conclude user agent was ABC based on version (`go-http-client` in our case).

Reply:
No problem - what do you have in mind? Would you rather send Golie version information via user-agent?

Comply with spec:
`User-Agent: <product> / <product-version> <comment>`
---
`User-Agent: Golie / v0.1.0`

I think we chatted yesterday about platform information.. is that still on the table?

Reply:
Thanks! `User-Agent: Golie / v0.1.0` is fine with me. But you can add as much details as you like.